README.md

```diff
@@ -26,69 +26,70 @@ Install the Docker generator:
 npm install -g generator-docker
 ```
 
-Run the generator in the same folder that your project is in:
+## Getting Started
+Once yo docker and VS Code are installed, here's the few steps to debug your first app
+### Node.js
+- From the root of your project, where your server.js file existts, open a terminal window
+```bash
+$ yo docker
+```
+- Answer the prompts, including your image name
+- Open the file you wish to debug and set a breakpoint
+- From VS Code, **[Command]** + **[P]** for the command palate
+- Enter: `task composeForDebug` This will compose your containers using the dockerTask.sh script
+- **[F5]** to start the node debugger within the container
+- As we use `nodemon -L --debug-brk`, you'll see the first line of server.js hit a breakpoint, so that you can debug the starting process of your app
```

Comment:
There isn't a breakpoint the debugger just breaks. Perhaps a better phrasing would be:
As we use `nodemon -L --debug-brk`, the debugger will break on the first line of your main file, so that you can debug the initialization of your app

---
In reply to: [68331947](https://github.com/Microsoft/generator-docker/pull/90#discussion_r68331947) [](ancestors = 68331947)